torchao/__init__.py

```diff
@@ -1,8 +1,28 @@
+from torchao.quantization import (
+    apply_weight_only_int8_quant,
+    apply_dynamic_quant,
+    change_linear_weights_to_int8_dqtensors,
+    change_linear_weights_to_int8_woqtensors,
+    change_linear_weights_to_int4_woqtensors,
+    swap_conv2d_1x1_to_linear,
+    autoquant,
+    change_linears_to_autoquantizable,
+    change_autoquantizable_to_quantized,
+)
 from . import dtypes
-from .quantization.quant_api import apply_dynamic_quant
-from .quantization.quant_api import apply_weight_only_int8_quant
 
 __all__ = [
```

Comment:
I don't think we should make all of these public right away. Like `apply_dynamic_quant` is a bit of a duplicate of `change_linear_weights_to_int8_dqtensors` and `swap_conv2d_1x1_to_linear` might be something we just want to do automatically instead of making it a toplevel API. 

Also dtypes is twice.

Is it possible to only add autoquant for now?